config/CMakeLists.txt

```diff
@@ -10,16 +10,20 @@ else()
   set(IMATH_ENABLE_API_VISIBILITY ON)
 endif()
 
+# Generate ImathConfig.h into src/Imath for use as a source header
```

Comment:
Apologies for the delay, I'm belatedly looking this over now. 

This is not right, we shouldn't be writing into the src directory. Two different builds running simultaneously in the same directory would overwrite the same `ImathConfig.h` file. A better solution is to add `${CMAKE_CURRENt_BINARY_DIR}`, where the second `configure_file` puts the `ImathConfig.h`, to the `target_include_directories` below.

Reply:
It's not going to work...

I had to generate `ImathConfig.h` in the `src` directory, as early as possible during configuration-phase, to have it included as a **source** file for Imath:
https://github.com/AcademySoftwareFoundation/Imath/blob/223cb1ce6e6655117575676566d6ccab31765a60/src/Imath/CMakeLists.txt#L22


(since CMake expects all of the public headers to be part of library's interface & if they're not, it simply wouldn't copy them to the `Headers` directory. I recall that we discussed this issue earlier.
The only other approach that comes to my mind is to copy the generated header into the `Headers` dir using a script, once builds are complete).

Reply:
We can, of course, leave the effort to the individual who's building the frameworks.